src/backend/expungeservice/expunger/expunger.py

```diff
@@ -48,7 +50,8 @@ def run(self):
         if self._open_cases():
             self.record.errors.append('All charges are ineligible because there is one or more open case.')
             return False
-
+        disposition_errors = self._build_record_errors_list_for_missing_or_unknown_dispositions(self.charges)
+        self.record.errors += disposition_errors
```

Comment:
You can inline this

`self.record.errors += self._build_record_errors_list_for_missing_or_unknown_dispositions(self.charges)`

But perhaps your reluctance suggests that maybe the function name itself should just be

`self.record.errors += self._build_disposition_errors(self.charges)`

Reply:
Yea that sounds good. I walked around in circles a few times here and got dizzy I guess, it happens.